parambokeh/__init__.py

```diff
@@ -29,6 +35,29 @@
     __version__ = '0.2.1-unknown'
 
 
+HOLOVIEWS_PROXY = """
+let HoloViews = {comms: {}, comm_status:{}, kernels:{}}
+window.HoloViews = HoloViews
+"""
+
+JS_CALLBACK = CustomJSCallback.js_callback
+
+
+def notebook_show(obj, doc, target):
+    """
+    Displays bokeh output inside a notebook and returns a CommsHandle.
+    """
+    mime = 'application/vnd.holoviews_exec.v0+json'
+    bokehcomm = bokeh.io.notebook.get_comms(target)
+    bokeh_script, bokeh_div, _ = bokeh.embed.notebook.notebook_content(obj, target)
+    publish_display_data(data={'text/html': encode_utf8(bokeh_div)})
+    publish_display_data(data={mime: '', 'application/javascript': bokeh_script},
+                         metadata={mime: {'id': target}})
```

Comment:
This needs to be the ``self.comm.id`` rather than the target.

Reply:
Thanks. This is what was needed to fix it.